SwiftWisdom/Core/Version/Bundle+Extensions.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  Bundle+Extensions.swift
+//  SwiftWisdom
+//
+//  Created by Paul Rolfe on 2/25/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import Foundation
+
+extension Bundle {
+
+    /**
+     Provides a way to access the current app version. If the app is a staging build it will return a SHA
+     if the build script is configured.
+     
+     - returns: String containing the current app version
+     */
+    var ip_appVersion: String? {
+        let key = "CFBundleShortVersionString"
+        let possibleVersion = infoDictionary?[key] as? String
+        var possibleSHA = ""
+
+        #if STAGING
```

Comment:
I don't think this preprocessor definition is safe to use in a library. Not everyone has a staging configuration and it may not be available in the library even if they do. It's actually all dependent on a build script to place the SHA there anyways. So for this one:
- Remove the `STAGING` conditional
- Make this part into a separate `public var ip_buildNumber`, without the parentheses we add into the string here and people can use it as they wish, without it being baked into `ip_appVersion`